content/docs/SUSHI/project/_index.md

```diff
@@ -107,22 +95,21 @@ Status (Default: draft): active
 Version (Default: 0.1.0): 2.0.0
 Initialize SUSHI project in C:\Users\shorty\dev\NewIG? [y/n]: y
 ```
-These values are used to generate a simple config.yaml file and a corresponding IG-Publisher-compatible project structure:
+These values are used to generate a simple **sushi-config.yaml** file and a corresponding IG-Publisher-compatible project structure:
 ```text
 NewIG
 ├── .gitignore
 ├── _genonce.bat     
 ├── _genonce.sh           
 ├── _updatePublisher.bat  
 ├── _updatePublisher.sh 
-├── _gencontinuous.bat 
-├── _gencontinuous.sh  
```

Comment:
No, I didn't add them to `sushi --init` because I wanted to keep `--init` as simple as possible, and in my view those scripts aren't really necessary for the flow most people using `--init` would be using.

Reply:
It will not be hard at all to add them back if we want them though.

Reply:
Nah.  I doubt people will miss them.

Reply:
I will say, `_gencontinuous` helped make me realize it is `gen-continuous` and therefore `genonce` is pronounced "jen-wons" and not "gen-awntz".